terraform/rancher/cluster/monitoring.tf

```diff
@@ -50,14 +50,42 @@ resource "rancher2_app_v2" "prometheus" {
   values        = <<-EOT
     cleanPrometheusOperatorObjectNames: true
     alertmanager:
+      config:
+        global:
+          resolve_timeout: 1m
+          slack_api_url: "${var.slack_webhook_url}"
+        route:
+          group_by: ['alertname', 'namespace']
+          group_wait: 30s
+          group_interval: 40s
+          repeat_interval: 30m
+          receiver: 'null'
+          routes:
+          - receiver: 'slack'
+            matchers:
+              - alertname =~ "Watchdog|InfoInhibitor"
+        receivers:
+        - name: 'null'
```

Comment:
Why do we need here receivers null?

Reply:
Just for notice that we have null and slack receivers, can be deleted.